lib/libsys/mmap.2

```diff
@@ -113,8 +113,45 @@ Pages may be read.
 Pages may be written.
 .It Dv PROT_EXEC
 Pages may be executed.
+.It Dv PROT_CAP
+CHERI capabilities may be read or written as dictated by
+.Dv PROT_READ
+and
+.Dv PROT_WRITE .
+.It Dv PROT_NO_CAP
+Respect the absence of
+.Dv PROT_CAP .
 .El
 .Pp
+On CHERI platforms, compatability is retained with unmodified POSIX
+programs by implying
+.Dv PROT_CAP
+based on the presence of
+.Dv PROT_READ
+and
+.Dv PROT_WRITE
+respectively, unless the underlying backing store can not safety support
+capabilities (e.g., a
+.Dv MAP_SHARED
+mapping of a file).
+If either of
+.Dv PROT_CAP
+or
+.Dv PROT_NO_CAP
+are set, then capability permissions will not be implied.
+When
+.Dv PROT_CAP
+is passed, at least one of
+.Dv PROT_READ
+and
+.Dv PROT_WRITE
+is required.
+On non-CHERI platforms the
+.Dv PROT_CAP
+and
+.Dv PROT_NO_CAP
+bits have no effect.
+.Pp
 In addition to these protection flags,
 .Fx
 provides the ability to set the maximum protection of a region allocated by
```

Comment:
Do we want any manpage language for the fact that PROT_MAX() must include an explicit PROT*_CAP flag if the regular permissions flags include one of those flags?